<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmMain</class>
 <widget class="QMainWindow" name="frmMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1231</width>
    <height>734</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout"/>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab1">
       <attribute name="title">
        <string>常用</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTabWidget" name="tabWidget_com">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="tabPosition">
           <enum>QTabWidget::North</enum>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_11">
           <attribute name="title">
            <string>base64转换</string>
           </attribute>
           <widget class="QLabel" name="base_image">
            <property name="geometry">
             <rect>
              <x>60</x>
              <y>260</y>
              <width>391</width>
              <height>321</height>
             </rect>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="lineWidth">
             <number>3</number>
            </property>
            <property name="text">
             <string>Image</string>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
           <widget class="QTextEdit" name="base_ret">
            <property name="geometry">
             <rect>
              <x>550</x>
              <y>10</y>
              <width>621</width>
              <height>571</height>
             </rect>
            </property>
           </widget>
           <widget class="QTextEdit" name="base_text">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>170</y>
              <width>511</width>
              <height>71</height>
             </rect>
            </property>
           </widget>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>0</y>
              <width>511</width>
              <height>161</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="2" column="1">
              <widget class="QPushButton" name="base_button_base2pic">
               <property name="text">
                <string>base64转图片</string>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QPushButton" name="base_button_download">
               <property name="text">
                <string>图片下载</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QPushButton" name="base_button_pic2base">
               <property name="text">
                <string>图片转base64</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QPushButton" name="base_button_text2base">
               <property name="text">
                <string>文字转base64</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QPushButton" name="base_button_base2text">
               <property name="text">
                <string>base64转文字</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QPushButton" name="base_button_clear">
               <property name="text">
                <string>清空数据</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QPushButton" name="base_button_choose">
               <property name="text">
                <string>选择文件</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0" colspan="2">
              <widget class="QLineEdit" name="base_text_file"/>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="QWidget" name="tab_12">
           <attribute name="title">
            <string>Consul管理</string>
           </attribute>
           <widget class="QTreeWidget" name="treeWidget_nodes">
            <property name="geometry">
             <rect>
              <x>540</x>
              <y>10</y>
              <width>631</width>
              <height>441</height>
             </rect>
            </property>
            <column>
             <property name="text">
              <string notr="true">1</string>
             </property>
            </column>
           </widget>
           <widget class="QTextEdit" name="textEdit_conusl_log">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>460</y>
              <width>1161</width>
              <height>141</height>
             </rect>
            </property>
           </widget>
           <widget class="QLabel" name="label_2">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>430</y>
              <width>93</width>
              <height>24</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>操作日志：</string>
            </property>
           </widget>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>40</y>
              <width>461</width>
              <height>301</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>18</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Consul Addr :</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEdit_consul_addr">
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>16</pointsize>
                </font>
               </property>
               <property name="text">
                <string>10.19.2.5:8500</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="3" column="0" colspan="3">
              <widget class="QPushButton" name="button_deregiest_all_off">
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>18</pointsize>
                </font>
               </property>
               <property name="text">
                <string>注销集群所有失效服务</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0" colspan="3">
              <widget class="QPushButton" name="button_deregiest_selected">
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>18</pointsize>
                </font>
               </property>
               <property name="text">
                <string>注销选中服务节点</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0" colspan="3">
              <widget class="QPushButton" name="button_consul_search">
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>18</pointsize>
                </font>
               </property>
               <property name="text">
                <string>查询集群所有节点</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QPushButton" name="pushButton">
            <property name="geometry">
             <rect>
              <x>440</x>
              <y>420</y>
              <width>75</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string>清空</string>
            </property>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab10">
       <attribute name="title">
        <string>视频监控</string>
       </attribute>
       <widget class="VideoWidget" name="vlcwidget" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>331</width>
          <height>341</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
       <widget class="QComboBox" name="vlc_comboBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>360</y>
          <width>251</width>
          <height>22</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="currentText">
         <string>rtsp://127.0.0.1:8090</string>
        </property>
       </widget>
       <widget class="QPushButton" name="vlc_pushButton">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>350</y>
          <width>81</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>播放</string>
        </property>
       </widget>
       <widget class="VideoWidget" name="vlcwidget_2" native="true">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>0</y>
          <width>361</width>
          <height>341</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
       <widget class="VideoWidget" name="vlcwidget_3" native="true">
        <property name="geometry">
         <rect>
          <x>780</x>
          <y>0</y>
          <width>391</width>
          <height>341</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
       <widget class="QComboBox" name="vlc_comboBox_2">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>360</y>
          <width>251</width>
          <height>22</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="currentText">
         <string>rtsp://127.0.0.1:8090</string>
        </property>
       </widget>
       <widget class="QPushButton" name="vlc_pushButton_2">
        <property name="geometry">
         <rect>
          <x>640</x>
          <y>350</y>
          <width>81</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>播放</string>
        </property>
       </widget>
       <widget class="QPushButton" name="vlc_pushButton_3">
        <property name="geometry">
         <rect>
          <x>1090</x>
          <y>350</y>
          <width>81</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>播放</string>
        </property>
       </widget>
       <widget class="QComboBox" name="vlc_comboBox_3">
        <property name="geometry">
         <rect>
          <x>830</x>
          <y>360</y>
          <width>251</width>
          <height>22</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="currentText">
         <string>rtsp://127.0.0.1:8090</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="video_textEdit">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>439</y>
          <width>1191</width>
          <height>201</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>400</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>操作日志：</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab5">
       <attribute name="title">
        <string>日历效果</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QCalendarWidget" name="calendarWidget1"/>
        </item>
        <item>
         <widget class="QCalendarWidget" name="calendarWidget2">
          <property name="gridVisible">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1231</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_O"/>
    <addaction name="separator"/>
    <addaction name="action_S"/>
    <addaction name="action_C"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="action_X"/>
    <addaction name="action_C_2"/>
    <addaction name="action_V"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="action_T"/>
    <addaction name="action_A"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
   <addaction name="menu_H"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>新建文件(&amp;N)</string>
   </property>
  </action>
  <action name="action_O">
   <property name="text">
    <string>打开文件(&amp;O)</string>
   </property>
  </action>
  <action name="action_S">
   <property name="text">
    <string>保存文件(&amp;S) </string>
   </property>
  </action>
  <action name="action_C">
   <property name="text">
    <string>关闭退出(&amp;C)</string>
   </property>
  </action>
  <action name="action_X">
   <property name="text">
    <string>剪切(&amp;X) </string>
   </property>
  </action>
  <action name="action_C_2">
   <property name="text">
    <string>复制(&amp;C)</string>
   </property>
  </action>
  <action name="action_V">
   <property name="text">
    <string>粘贴(&amp;V)</string>
   </property>
  </action>
  <action name="action_T">
   <property name="text">
    <string>帮助文档(&amp;T)</string>
   </property>
  </action>
  <action name="action_A">
   <property name="text">
    <string>关于(&amp;A)</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VideoWidget</class>
   <extends>QWidget</extends>
   <header>videowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
